site/content/docs/api-checks/_index.md

```diff
@@ -13,64 +13,107 @@ aliases:
+![Screenshot of the create API check page](/docs/images/api-checks/overview-create-check.png)
 
-![http request](/docs/images/api-checks/overview-http.png)
+### Name and tags
+
+A meaningful name will not only help you and others identify your checks within Checkly, but it will help provide a better alerting experience if your checks fall into an alert state.
+
+Tags can relate your checks together. They also determine which checks are shown on your [dashboards](/docs/dashboards/).
+
+
+### HTTP request
+
+This is where you configure the API endpoint. The most important part of your API check is its URL and method. Optionally, you can define body data, headers, query parameters and authentication for the API check. These [request settings](request-settings) determine the actions taken by the API and the response sent back. 
+
+Endpoints can be accessed and manipulated through [HTTP methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods). Not all methods are valid for all endpoints.
 
 You can import a cURL command, Swagger.json or OpenAPI specification here too. 
 
-**3. Setup and teardown scripts**
+### Setup and teardown scripts
 
-Setup scripts allow you to do last minute processing of test data and request options. The scripts execute before any requests are made.
-Teardown scripts are run after the HTTP request has finished, but before any assertions are validated. With a maximum execution time of 10 seconds, these scripts are really useful for things like signing HMAC requests, requesting tokens, setting up or cleaning up test data and scrubbing sensitive response data for regulatory reasons.
+Setup scripts allow you to do last minute processing of test data and request options. These scripts execute before any requests are made. Teardown scripts are run after the HTTP request has finished, but before any assertions are validated. With a maximum execution time of 10 seconds, these scripts are useful for things like signing HMAC requests, requesting tokens, setting up or cleaning up test data and scrubbing sensitive response data for regulatory reasons.
 
 [Read more about setup and teardown scripts](setup-teardown-scripts/)
-![setup and teardown scripts](/docs/images/api-checks/overview-scripts.png)
 
-**4. Response time limits**
+### Response time limits
 
-Our [time limits](limits) are customisable, sometimes API's can be slow, but not broken. We call that **degraded**. We let you specify when an API check should be marked as **degraded** and when it should be marked as **failed**.
-![response times](/docs/images/api-checks/overview-response-time.png)
+Sometimes APIs can be slow, but not broken. We call this **degraded**. You can set [response time limits](/docs/api-checks/limits/) to specify when an API check should be marked as **degraded** and when it should be marked as **failed**.
```

Comment:
@guolau we're only using bold text in this section. Can we either stick to plain text or highlight this differently, maybe with ``?